Automattic/sensei
# Fix warnings related to email html

Resolves #6573 

## Proposed Changes
* Use libxml internal errors instead of firing system warnings.
* ~~Add /vendor dir to archive.~~

## Testing Instructions

0. Add `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );` to `sensei-lms.php`
1. Go to composer.json and add `"!/vendor"` to `archive > exclude`.
2. `npm i && composer install`
3. `npm run build`
4. Go to Jurassic and create a website.
5. Install plugin from generated archive in step 3.
6. Create and publish a course with lessons.
7. Add a student.
8. As a student, go to the frontend, take a course and complete all the lessons.
9. Make sure there were no erros.
10. Make sure you got all the expected emails (depends on the state of code you used for creating the archive).

## Pre-Merge Checklist
